adventure-crew-game/Assets/Scripts/UI/Dialogue/Dialogue Config.asset

```diff
@@ -15,7 +15,35 @@ MonoBehaviour:
   characters:
   - charName: Griffith
     dialogue:
-    - Hi there
-    - My name is Griffith Berserk
+    - You fought well, but it is time to accept your defeat. You seem resourceful,
+      if you wish to continue living you'll join us.
+    - I am Griffith, leader of the Band of the Hawk.
+    - Firstly, let me introduce you to your squad.
+    charBox: {r: 0.08627451, g: 0.2784314, b: 0.54901963, a: 1}
+    charArt: {fileID: 21300000, guid: aec913b28bfc6ab4a86163bcf7861cef, type: 3}
+  - charName: Griffith
+    dialogue:
+    - Here you can view the status of all of your soldiers.
```

Comment:
I'd add something to the effect of "Hover your mouse over each one to get a breakdown of their abilities."